src/extensionConflicts.ts

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved..
+ *  Licensed under the MIT License. See LICENSE in the project root for details.
+ *--------------------------------------------------------------------------------------------*/
+import { commands, Extension, extensions, window } from 'vscode';
+
+// A set of VSCode extension ID's that conflict with our extension
+const conflictingIDs = [
+  'haaaad.ansible',
+  'sysninja.vscode-ansible-mod',
+  'tomaciazek.ansible',
+  'vscoss.vscode-ansible',
+  'zbr.vscode-ansible',
+];
+
+// A set of VSCode extension ID's that are currently uninstalling
+const uninstallingIDs = new Set();
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function isExtensionPresent(obj: any): obj is Extension<any> {
+  return typeof obj !== 'undefined' && !uninstallingIDs.has(obj.id);
+}
+
+/**
+ * Get all of the installed extensions that currently conflict with VSCode-YAML
```

Comment:
Replace vscode-yaml with redhat.ansible extension?

Reply:
Thanks! I fixed that.